apiserver/facades/client/secrets/secrets.go

```diff
@@ -200,6 +208,34 @@ func (s *SecretsAPI) ListSecrets(ctx context.Context, arg params.ListSecretsArgs
 	return result, nil
 }
 
+func tagFromSubject(access secretservice.SecretAccessor) (names.Tag, error) {
```

Comment:
We don't want to put this kind of helpers in common anymore, right?

Reply:
For a few simple lines of code only used in 2 places, it's ok to repeat I think. We can see how we like it...